--- /var/vcap/packages/cloud_controller/cloud_controller/app/models/app.rb	2012-01-30 20:24:18.000000000 +0000
+++ /var/vcap/packages/cloud_controller/cloud_controller/app/models/app_new.rb	2012-03-13 23:24:19.214790495 +0000
@@ -23,8 +23,8 @@
 
   AppStates = %w[STOPPED STARTED]
   PackageStates = %w[PENDING STAGED FAILED]
-  Runtimes = %w[ruby18 ruby19 java node node06 php erlangR14B02 python26]
-  Frameworks = %w[sinatra rails3 java_web spring grails node php otp_rebar lift wsgi django unknown]
+  Runtimes = %w[ruby18 ruby19 java node php erlangR14B02 python26 aspdotnet40]
+  Frameworks = %w[sinatra rails3 java_web spring grails node php otp_rebar lift aspdotnet wsgi django unknown]
 
   validates_presence_of :name, :framework, :runtime
 
